README.md

```diff
@@ -1,2149 +1,92 @@
-
-Next, you will have access to the following parameters and methods:
-
-Name           | Description
--------------- | -------------
-`config`       | Returns parsed Testplane config.
-`events`       | Returns Testplane events on which you can subscribe to.
-`errors`       | Errors which Testplane may return.
-`intercept`    | Method to intercept Testplane's events.
-`run`          | Starts running tests. By default run all tests from the config. Can also run only the specified tests.
-`addTestToRun` | Adds test to the current run.
-`readTests`    | Starts reading tests. By default read all tests from the config. Can also read only the specified tests.
-`isFailed`     | Returns `true` or `false` depending on whether there has been an error or a test fail while running tests.
-`isWorker`     | Returns `true` or `false` depending on whether you call the method in one of the workers or in the master process.
-`halt`         | Abnormal termination of the test run in case of a terminal error.
-
-### config
-
-Returns parsed Testplane config. Useful when you need to read some field from the config.
-
-```js
-// create testplane instance
-console.log('plugins:', testplane.config.plugins);
-```
-
-### events
-
-Returns Testplane events on which you can subscribe to. Useful when you need to subscribe to a specific event.
-
-```js
-// create testplane instance
-testplane.on(testplane.events.INIT, async () => {
-    console.info('INIT event is being processed...');
-});
-```
-
-### errors
-
-Testplane may return errors of the following type:
-
-- [CoreError](#coreerror)
-- [CancelledError](#cancellederror)
-- [ClientBridgeError](#clientbridgeerror)
-- [HeightViewportError](#heightviewporterror)
-- [OffsetViewportError](#offsetviewporterror)
-- [AssertViewError](#assertviewerror)
-- [ImageDiffError](#imagedifferror)
-- [NoRefImageError](#norefimageerror)
-
-#### CoreError
-
-A `CoreError` is returned if the browser fails to calibrate a blank page (about:blank). The error contains the following message:
-
-```
-Could not calibrate. This could be due to calibration page has failed to open properly
-```
-
-#### CancelledError
-
-The `CanceledEror` error is returned if the [halt](#halt) command terminates abnormally. The error contains the following message:
-
-```
-Browser request was cancelled
-```
-
-#### ClientBridgeError
-
-A `ClientBridgeError` is returned when JavaScript injection on the client (browser) side fails. Testplane injects the code using the [execute](https://webdriver.io/docs/api/browser/execute/) WebDriverIO command. The error contains the following message:
+3. Launch GUI:
 
-```
-Unable to inject client script
-```
+    ```shell
+    npx testplane gui
+   ```
+   
+4. Try running the test, which will initially fail because a reference image for the heading is missing. You can accept the diff and re-run the test, it will then pass.
```

Comment:
maybe, for the initial run, we only need to check a few scenarios without any screenshots to confirm that the test is correct.

And then we will add screenshot assert